app/resources/api/v2/plate_purpose_resource.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API representation of Plate
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class PlatePurposeResource < BaseResource
+      model_name 'PlatePurpose'
+
+      # This resource is similar to PurposeResource but it was created to
+      # migrate the registration of plate purposes done by the Limber rake
+      # task config:generate from API version 1 to API version 2.
```

Comment:
Why was it necessary to create PlatePurposeResource? Did you consider adding to PurposeResource instead?

Reply:
Purpose resource is "included" in other resources and used by Limber. I wanted to avoid make changes in Purpose resource. The use-case for having a plate purpose resource at this time is migrating plate purpose creation to SS API v2  to support passing a custom attribute 'asset_shape' as a name. When we start working on DPL-919 and after deciding whether to expose an AssetShape resource instead of passing name, we can think about unifying all Purpose, plate purpose and tube purposes in v2.